lib/app.rb

```diff
@@ -44,7 +37,11 @@ class App < Sinatra::Base
   # Search for Manifests
   get '/api/:version/manifest/search' do
     content_type :json
-    Manifest.all.to_json
+    if !params[:q] && !params[:aq]
```

Comment:
what about giving this param a more verbose name like `query`? (not sure what `aq` means, but a longer name might help with that as well)